packages/bsky/src/hydration/hydrator.ts

```diff
@@ -219,11 +219,12 @@ export class Hydrator {
     ctx: HydrateCtx,
   ): Promise<HydrationState> {
     const [lists, listViewers, labels] = await Promise.all([
-      this.graph.getLists(uris),
+      this.graph.getLists(uris, ctx.includeTakedowns),
       ctx.viewer ? this.graph.getListViewerStates(uris, ctx.viewer) : undefined,
       this.label.getLabelsForSubjects(uris, ctx.labelers),
     ])
 
+    // TODO: Check why the takedown label is not coming through here when list has been takendown
```

Comment:
Is this still an issue?

Reply:
(Tiiiny consistency thing)
```suggestion
    // @TODO Check why the takedown label is not coming through here when list has been takendown
```

Reply:
nope, @dholms helped me figure out the label issue.